Formula/o/openjazz.rb

```diff
@@ -20,11 +19,11 @@ class Openjazz < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "a50f5226a5b83bd7f316b4f1546124fcf1b84cb99a18909a6e9551662a5a17b1"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "libmodplug"
```

Comment:
`libmodplug` should be removed.
I do not know how SDL finding works with homebrew, but I think `pkg-config` could be removed as well.

Reply:
`cmake` should be able to find SDL on its own, so, yeah, `pkg-config` could probably be removed.